docker-compose.prod.yml

```diff
@@ -31,7 +31,7 @@ services:
     depends_on:
       - web
   graphql-engine:
-    image: hasura/graphql-engine:v1.0.0-beta.10
+    image: hasura/graphql-engine:v1.0.0-beta.10.cli-migrations
```

Comment:
Could we merge https://github.com/zagrajmy/zm-backend/pull/16 instead of using beta version?

Reply:
@lprasol

Reply:
There's a changelog. https://github.com/hasura/graphql-engine/releases